src/patternfly/components/Chip/chip.scss

```diff
@@ -1,31 +1,17 @@
 .pf-c-chip {
+  --pf-c-chip--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-chip--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip--PaddingBottom: var(--pf-global--spacer--form-element);
   --pf-c-chip--PaddingLeft: var(--pf-global--spacer--sm);
   --pf-c-chip--BackgroundColor: var(--pf-global--Color--light-100);
-  --pf-c-chip--BorderColor: var(--pf-global--secondary-color--100);
+  --pf-c-chip--BorderColor: var(--pf-global--BackgroundColor--200);
```

Comment:
should this be `--pf-global--BorderColor--300`?

Reply:
oh yep nice catch